python-checks/src/main/resources/org/sonar/l10n/py/rules/python/S6537.html

```diff
@@ -0,0 +1,25 @@
+<p>Octal escapes are included in a limited form. If the first digit is a 0, or if there are three octal digits, it is considered an octal escape.
+Otherwise, it is a group reference. As for string literals, octal escapes are always at most three digits in length.</p>
```

Comment:
After re-reading this part, I think it would be better for it to be a short description of the rule (octal escapes should not be used), rather than explaining what an octal escape it. I'll leave a comment/suggestion on the rspec PR, but just to let you know that I think we should update this.